pkg/lambdacomponents/defaults.go

```diff
@@ -36,6 +38,13 @@ func Components() (
 ) {
 	var errs error
 
+	extensions, err := component.MakeExtensionFactoryMap(
+		sigv4authextension.NewFactory(),
```

Comment:
This extension factory  map needs to be added to the returned `factories` struct at the end of the function.

Reply:
Fixede